### ISSUE TITLE
this.config not set

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ngocanh7948github/gitiles#33]
```
What steps will reproduce the problem?
1. deploy application
2. browse first page


What is the expected output?
working index page

What do you see instead?
java.lang.NullPointerException: cfg


What version of the product are you using?
commit 076b1bf9bb40bd653ff2642903c7efdd8bb36d32

 On what operating system?
x86_64-freebsd

Please provide any additional information below.
com.google.gitiles.GitilesFilter.setDefaultFields fails to set this.config if 
it is null.


```

Original issue reported on code.google.com by `qui...@gmail.com` on 24 Oct 2013 at 4:47

**Comment by GoogleCodeExporter** [comment]
```
The attached patch fixes the issue.
```

Original comment by `qui...@gmail.com` on 24 Oct 2013 at 4:48

Attachments:
- [0001-Set-this.config-at-initialization.patch](https://storage.googleapis.com/google-code-attachments/gitiles/issue-33/comment-1/0001-Set-this.config-at-initialization.patch)

**Comment by GoogleCodeExporter** [comment]
```
This is with Jetty 8.1.11.
```

Original comment by `qui...@gmail.com` on 24 Oct 2013 at 4:50

**Comment by GoogleCodeExporter** [comment]
```
This appears to have been fixed in b5069825016ddef40875f40126fd71193bbb7f43
```

Original comment by `qui...@gmail.com` on 12 Nov 2014 at 2:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `sop@google.com` on 24 Jun 2015 at 5:39
- Changed state: **Fixed**
